client/src/chess/auth/auth.component.js

```diff
@@ -23,22 +31,21 @@ function AuthController($location, auth) {
     auth.login(data)
     .then(
       function (message) {
-        $location.path( "/chess" );
-        alert('Welcome' + ' ' + data.username)
+        $location.path( "/players-room" );
       },
     showError)
   }
   function submitRegisterForm() {
     var data = {
+      email: $ctrl.email,
       username : $ctrl.username,
       password : $ctrl.password
     };
     auth.register(data)
     .then(
       function (message) {
-        alert(message);
+        $location.path( "/players-room" );
```

Comment:
make it configurable: change AuthFactory to provider, and add ability to change the redirection target on successfull registrations (and login too)